lib/grammar2.pegjs

```diff
@@ -72,7 +71,13 @@ ctrl = ifctrl
      / forctrl
      / returnctrl
 
-ifctrl     = "if" _ c:innerstmt _ b:block { return p.parseIf(c, b) }
+ifctrl = "if" _ c:innerstmt __ b:block ei:(__ elseifcont)* {
+  ei = ei.map(function (pair) { return pair[1] })
+  return p.parseIf(c, b, ei)
+}
+// Continuations of the if control with else-ifs
+elseifcont = "else" __ "if" _ c:innerstmt __ b:block { return p.parseIf(c, b, null) }
```

Comment:
`__` is white-space, newlines, and comments. `_` is just white-space (so just space and tab IIRC).